docs/guides/flight-recorder.md

```
@@ -304,12 +306,16 @@ Fields:
 
 ## Hash Chain
 
-The hash covers all entry fields joined with null-byte separators:
+Each schema version has a frozen field projection joined with null-byte separators. V1 uses:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)